src/status_im/chat/handlers/faucet.cljs

```diff
@@ -0,0 +1,60 @@
+(ns status-im.chat.handlers.faucet
+  (:require [re-frame.core :refer [dispatch]]
+            [status-im.utils.handlers :refer [register-handler] :as u]
+            [status-im.utils.utils :refer [http-get]]
+            [status-im.utils.random :as random]
+            [status-im.constants :refer [console-chat-id
+                                         text-content-type]]
+            [status-im.i18n :refer [label]]
+            [taoensso.timbre :as log]
+            [goog.string :as gstring]
+            goog.string.format))
+
+(def faucets
+  [{:name     "http://faucet.ropsten.be:3001"
+    :type     :api
+    :api-url  "http://faucet.ropsten.be:3001/donate/0x%s"}
+   {:name       "https://zerogox.com/ethereum/wei_faucet"
+    :type       :prefill
+    :prefill-js "document.getElementById('edit-ethereum-address').value='%s';"}])
+
+(defn faucet-by-name [faucet-name]
+  (->> faucets
+       (filter #(= (:name %) faucet-name))
+       (first)))
+
+(defn received-message [content]
+  (dispatch [:received-message
+             {:message-id   (random/id)
+              :content      content
+              :content-type text-content-type
+              :outgoing     false
+              :chat-id      console-chat-id
+              :from         console-chat-id
+              :to           "me"}]))
+
+(defmulti open-faucet (fn [_ _ {:keys [type]}] type))
+
+(defmethod open-faucet :api
+  [faucet-name current-address {:keys [api-url]}]
+  (let [api-url (gstring/format api-url current-address)]
+    (http-get api-url
+              #(received-message (label :t/faucet-success))
+              #(received-message (label :t/faucet-error)))))
+
+(defmethod open-faucet :prefill
+  [faucet-name current-address {:keys [prefill-js]}]
+  (let [prefill-js  (gstring/format prefill-js current-address)
+        web-view-js (gstring/format "window.onload = function(){ %s };" prefill-js)]
```

Comment:
No, it is not possible, unfortunately. There is an extra form parameter and they check this paramer when you send the form.
<img width="556" alt="screen shot 2016-12-08 at 13 50 55" src="https://cloud.githubusercontent.com/assets/911127/21007352/95ec9e78-bd4d-11e6-90a5-04b075d90675.png">

For example, this query works:
```curl --data "ethereum_address=24c4d1b07da971c037d3db3f3837fd840037acd6&form_build_id=form-V-rIjQnyT1eErKXbnmJwW4lVr7SlQg6PenYPrDBQmC0&form_id=ethereum_wei_faucet_form" https://zerogox.com/ethereum/wei_faucet```

but this one doesn't work
```curl --data "ethereum_address=24c4d1b07da971c037d3db3f3837fd840037acd6&form_id=ethereum_wei_faucet_form" https://zerogox.com/ethereum/wei_faucet```